### PR TITLE
Handle ground class input in find_holes, avoid crash on zero-feature tiles in pc_repair_man

### PR DESCRIPTION
**Pull request by plimkilde**

### CODE REVIEW

File: qc/find_holes.py [hunk 1]
```diff
@@ -63,7 +63,7 @@
 parser.add_argument(
     "-class",
     type=int,
-    default=5,
+    default=constants.terrain,
     help="Specify ground class in reference pointcloud. Defaults to 5 (dhm-2007).",
```
Comment (kbevers) [inline qc/find_holes.py]: Remove the parenthesis. Not valid anymore.

File: qc/find_holes.py [hunk 2]
```diff
@@ -199,7 +199,7 @@ def main(args):
 
     # should be as a terrain grid - but problems with high veg on fields!!!
     pc = pointcloud.fromAny(pargs.las_file).cut_to_class(cut_to)
-    pc_ref = pointcloud.fromAny(pargs.ref_data).cut_to_class(5)
+    pc_ref = pointcloud.fromAny(pargs.ref_data).cut_to_class(constants.terrain)
```
Comment (kbevers) [inline qc/find_holes.py]: use `pargs.class` here instead. Otherwise overriding the default value will have not effect.

File: qc/find_holes.py [hunk 3]
```diff
@@ -199,7 +199,9 @@ def main(args):
 
     # should be as a terrain grid - but problems with high veg on fields!!!
     pc = pointcloud.fromAny(pargs.las_file).cut_to_class(cut_to)
-    pc_ref = pointcloud.fromAny(pargs.ref_data).cut_to_class(5)
+    
+    # "class" is a reserved keyword...
+    pc_ref = pointcloud.fromAny(pargs.ref_data).cut_to_class(getattr(pargs, 'class'))
```
Comment (kbevers) [inline qc/find_holes.py]: `pargs.class` is possible here and the same pattern is used throughout the code base. It is simpler to read and write than `getattr(pargs, 'class'))`.

Comment (plimkilde) [reply qc/find_holes.py]: Tried using pargs.class. It causes a SyntaxError to be raised.

Comment (plimkilde) [reply qc/find_holes.py]: Going with the ```dest=``` option to avoid breaking the backward compatibility of configurations for now.

**Pull request merged**